### ISSUE TITLE
refresh stakepool information before purchasing tickets in case pool fees have changed

### ISSUE THREAD

**Issue by jolan** [decred/Paymetheus#234]
I looked and I don't think this is being done already but if I'm wrong, please close this issue.

Request from @behindtext.

**Comment by fainges** [comment]
No, we've just had a guy who left PM open for a while then tried to buy tickets. It said the given expiry was before the current block. This will need to be fixed as it means people will not get the expiry they think they're getting.
